app/src/pages/experiment/ExperimentCompareTable.tsx

```diff
@@ -683,33 +685,46 @@ function ExperimentRowActionMenu(props: {
         e.stopPropagation();
       }}
     >
-      <ActionMenu
-        buttonSize="compact"
-        align="end"
-        onAction={(firedAction) => {
-          const action = firedAction as ExperimentRowAction;
-          switch (action) {
-            case ExperimentRowAction.GO_TO_EXAMPLE: {
-              return navigate(`/datasets/${datasetId}/examples/${exampleId}`);
-            }
-            default: {
-              assertUnreachable(action);
-            }
-          }
-        }}
-      >
-        <Item key={ExperimentRowAction.GO_TO_EXAMPLE}>
-          <Flex
-            direction={"row"}
-            gap="size-75"
-            justifyContent={"start"}
-            alignItems={"center"}
-          >
-            <Icon svg={<Icons.ExternalLinkOutline />} />
-            <Text>Go to example</Text>
-          </Flex>
-        </Item>
-      </ActionMenu>
+      <DialogTrigger>
+        <Button
+          size="S"
+          leadingVisual={<Icon svg={<Icons.MoreHorizontalOutline />} />}
+        />
+        <Popover>
```

Comment:
we probably could just make our own action menu component. Not blocking

Reply:
Yes, we really need one. Still lots of those to port, and they break new dialogs